### DISCUSSION TITLE
How to add style classes when hovering over a sibling element?

### DISCUSSION THREAD

**Discussion by MonaAghili** [tailwindlabs/tailwindcss#14267] [Help] [answered]
As shown in the picture, I want the element with the `content` class to become `visible` when hovering over the element with the `trigger` class. Is there a way to achieve this?
In this example, we can only show the `content` when hovering over the `wrapper` element.
Unfortunately, it seems Tailwind doesn't support sibling selectors (like `+`).

```
<div class="wrapper relative w-fit group">
  <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ea officiis quia sit dicta. A inventore sed cum eligendi. Eos <button class="trigger cursor-pointer">Trigger</button></p>

  <div class="content absolute w-full invisible group-hover:visible">
    Lorem, ipsum dolor sit amet consectetur adipisicing elit.
  </div>
</div>
```
![Untitled Diagram drawio](https://github.com/user-attachments/assets/84866e17-f703-427b-a868-fdbf064c2407)

**Comment by cannap** [comment, marked as answer]
hi  this? via "peer" 

https://play.tailwindcss.com/tuqFp9QtQR
